packages/lib-classifier/src/plugins/drawingTools/components/Point/Point.stories.js

```diff
@@ -1,25 +1,143 @@
+import { Provider } from 'mobx-react'
+import { Grommet } from 'grommet'
 import zooTheme from '@zooniverse/grommet-theme'
-import React, { Component, createRef, forwardRef } from 'react'
-import sinon from 'sinon'
-import { PointTool } from '@plugins/drawingTools/models/tools'
-import { Mark }from '@plugins/drawingTools/components'
-import Point from './Point'
+import asyncStates from '@zooniverse/async-states'
+import cuid from 'cuid'
+import SingleImageViewer from '@viewers/components/SingleImageViewer'
+import ClassificationStore from '@store/ClassificationStore'
+import SubjectViewerStore from '@store/SubjectViewerStore'
+import DrawingTask from '@plugins/tasks/DrawingTask/models/DrawingTask'
+import { DrawingTaskFactory, ProjectFactory, SubjectFactory, WorkflowFactory } from '@test/factories'
+
+const subject = SubjectFactory.build({
+  locations: [
+    { 'image/jpeg': 'http://placekitten.com/500/300' }
+  ]
+})
+
+const project = ProjectFactory.build()
+const workflow = WorkflowFactory.build()
+const drawingTaskSnapshot = DrawingTaskFactory.build({
+  instruction: 'Draw a point',
+  taskKey: 'T1',
+  tools: [{
+    color: zooTheme.global.colors['drawing-red'],
+    type: 'point'
+  }],
+  type: 'drawing'
+})
+
+const subTasksSnapshot = [
+  {
+    instruction: 'Name your favourite fruit.',
+    taskKey: 'T0.0',
+    type: 'text'
+  },
+  {
+    answers: [{ label: "yes" }, { label: "no" }],
+    help: "",
+    question: "Is it tasty?",
+    taskKey: 'T0.1',
+    type: 'single'
+  },
+  {
+    answers: [{ label: "cat" }, { label: "dog" }, { label: "bird" }],
+    help: "",
+    question: "Select your favourite animals.",
+    taskKey: 'T0.2',
+    type: 'multiple'
+  }
+]
+
+function setupStores ({ activeMark, subtask }) {
+  if (subtask) {
+    drawingTaskSnapshot.tools[0].details = subTasksSnapshot
+    drawingTaskSnapshot.subTaskVisibility = true
+    // should think of a better way to do this for the story
+    // this is a rough approximation of what the positioning is like now
+    drawingTaskSnapshot.subTaskMarkBounds = {
+      x: 162.5,
+      y: 162.5,
+      width: 0,
+      height: 0,
+      top: 0,
+      right: 0,
+      bottom: 0,
+      left: 0
+    }
+  } 
+
+  const drawingTask = DrawingTask.create(drawingTaskSnapshot)
+  drawingTask.setActiveTool(0)
+  const point = drawingTask.activeTool.createMark()
+  point.move({ x: 100, y: 100 })
+
+  const mockStores = {
+    classifications: ClassificationStore.create(),
+    subjects: {
+      active: subject
+    },
+    subjectViewer: SubjectViewerStore.create(),
+    workflows: {
+      active: { id: cuid() }
+    },
+    workflowSteps: {
+      activeStepTasks: [drawingTask]
+    }
+  }
+
+  mockStores.classifications.createClassification(subject, workflow, project)
+  mockStores.workflowSteps.activeStepTasks[0].updateAnnotation()
+  if (activeMark) {
+    mockStores.workflowSteps.activeStepTasks[0].setActiveMark(point.id)
+  }
+
+  if (subtask) {
+    mockStores
```

Comment:
What is happening here? Is this conditional needed in an upcoming PR?

Reply:
Oh this is cruft I think left over when I was trying to figure out how to implement the story. Good catch